hc/accounts/views.py

```diff
@@ -196,6 +196,21 @@ def profile(request):
                 profile.team_name = form.cleaned_data["team_name"]
                 profile.save()
                 messages.success(request, "Team Name updated!")
+        elif "update_priority" in request.POST:
+            if not profile.team_access_allowed:
+                return HttpResponseForbidden()
+            form = UpdatePriorityForm(request.POST)
+            if form.is_valid():
+                email = form.cleaned_data["email"]
+                user = User.objects.get(email=email)
+                member = Member.objects.filter(team=profile, user=user).first()
+            if member.priority == "LOW":
```

Comment:
@Sebuliba-Adrian  what does this if statement do?

Reply:
The if statement is meant to implement the priority toggle logic from either a "LOW" priority to a "HIGH" priority or vise verse whenever the Team Owner clicks the priority button.

Reply:
@WNjihia  The priority is LOW by default.